ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ntp.yml

```diff
@@ -5,4 +5,13 @@ ntp_server:
     interface: lo1
   nodes:
     - 10.1.1.1
-    - 10.1.1.2
\ No newline at end of file
+    - 10.1.1.2
+  ntp_authenticate: true
+
+  ntp_authentication_keys:
+    10:
+      hash_algorithm: "md5"
+      key:  "356708192346f678a878d6e99889767a"
+  
+  ntp_trusted_keys:
```

Comment:
`ntp_trusted_keys` should be a simple string containing multiple keys and not a list.

example of possible values:

```yaml
 ntp_trusted_keys: "10"
 ntp_trusted_keys: "10-12,15"
 ntp_trusted_keys: "10,12,15"
```